src/Main.java

```diff
@@ -1,5 +1,24 @@
+import lesson6tasks.Circle;
+import lesson6tasks.Dog;
+import lesson6tasks.Point;
+
 public class Main {
     public static void main(String[] args) {
-        System.out.println("Hello world!");
+        Rectangle rectangleFirst = new Rectangle(12,23);
+        Circle circleFirst = new Circle(12,3.14);
+        Dog dogFirst = new Dog("Retriever",2,"Doggy");
+
+        System.out.println(rectangleFirst.calculateArea(12,23));
+        System.out.println(rectangleFirst.calculatePerimeter(12,23));
```

Comment:
від імені rectangleFirst викликається метод підрахунку площі, на цьому ж обєкті. Ніяких параментрів не потрібно 
```suggestion
        System.out.println(rectangleFirst.calculateArea());
        System.out.println(rectangleFirst.calculatePerimeter());
```